packages/omgx/wallet-frontend/src/actions/networkAction.js

```diff
@@ -153,10 +153,10 @@ export function resetApprove (value, currency, approveContractAddress, contractA
   );
 }
 
-export function depositErc20 (value, currency, gasPrice) {
+export function depositErc20 (value, currency, gasPrice, currencyL2) {
```

Comment:
does is mean to say?
L1Address === currency?
L2Address === currencyL2?

Reply:
Yes, that's right.